.gitignore

```diff
@@ -1,5 +1,4 @@
 .DS_Store
 node_modules/
 npm-debug.log
-docs/_site/
-dist/
\ No newline at end of file
+docs/_site/
```

Comment:
can you restore this as per master and remove:

-dist files
-examples/package.json
-package.json version to the one on master

sorry for the inconvenience!